### ISSUE TITLE
When amending particle setting block size is shown as invalid when it is not.

### ISSUE THREAD

**Issue by marktaylor46** [gantry/gantry5#1025]
This is best illustrated through example:
Take Isotope template and look at the home layout copyright section.
Notice that there are three particles each of 33.3% width.
delete the middle on and you then have 50% 50%.
Change the first one to 33.3% width - the other one change to 66.7% width and SAVE.
Now edit the settings for the second particle - you won't be able to apply any change because the "66.7" on the block tab is being reported as an "invalid field".

I gave the verbose example because this is a real-life example of what members do - I know I could have just started with two particles.

So, basically the "66.7" is perfectly valid and yet is being flagged as an error.
![2015-11-24_13-57-38](https://cloud.githubusercontent.com/assets/11683218/11368921/3b1ed7be-92b4-11e5-9db1-04b60f17edad.png)
![2015-11-24_14-05-03](https://cloud.githubusercontent.com/assets/11683218/11368954/6a986230-92b4-11e5-9188-d2bcd641805d.png)

**Comment by w00fz** [comment]
Technically 66.7 is not valid at all. You can never get 66.7 if you resize via drag and drop.
When you have `33.3 | 33.3 | 33.3`, if you try to resize via UI, you will see that they get automatically rearranged to `33 | 33 | 34`, this is because there are a very few cases where decimal are necessary, but the entire layout, aside from the exceptions of equalized blocks such as 3, is meant to be round.

The fact that you can change to `33.3` any block via the Block tab is a known thing and kind of a limitation. It is very hard to determine case by case when you are allowed to do 33.3 and when not, but it should never invalidate the value of `33.3`. It needs to be always allowed to be typed in, otherwise if you were to edit a legit 33.3, it would invalidate and you wouldn't be able to change anything in the form.

Since the Block Size field is simply an additional location meant for those who want to manually edit that value, rather than using drag & drop handles, we leave to the common sense how to treat those values. There is some validation in place but it's not capable of determine all possible cases.

It is always advised to handle it via drag & drop, has it is more accurate and is capable of auto adjusting when needed.

In that case, when you are left with `50 | 50`, you will never be able to get to `33.3` again, if you were drag and dropping. By manually inputting it into the field you are "cheating" the system.

I wish it was easier to handle it better on code but it is very complicated dealing with these exceptions. I don't think it's worth spending time into fixing this.
